crates/bevy_ecs/src/reflect/component.rs

```diff
@@ -277,7 +277,8 @@ impl<C: Component + Reflect + TypePath> FromType<C> for ReflectComponent {
             },
             apply: |mut entity, reflected_component| {
                 if !C::Mutability::MUTABLE {
-                    panic!("This component is immutable, you cannot modify it through reflection");
+                    let name = disqualified::ShortName::of::<C>();
```

Comment:
I'm going to give you the same nit you mentioned to me :p

We prefer to add an import instead of using the FQN so that we can see, at a glance, what crates are being pulled in for this module.

Reply:
My hubris! It hurts!